File: Easy/Python/1773. Count Items Matching a Rule.py
class Solution(object):
    def countMatches(self, items, ruleKey, ruleValue):
        count = 0

        for item in items:
            if ((ruleKey == "type" and ruleValue == item[0])
            or (ruleKey == "color" and ruleValue == item[1])
            or (ruleKey == "name" and ruleValue == item[2])):
                count += 1

        return count

File: Easy/JavaScript/1773. Count Items Matching a Rule.js
var countMatches = function(items, ruleKey, ruleValue) {
    let count = 0;

    items.forEach(item => {
        if((ruleKey === "type" && ruleValue === item[0]) ||
           (ruleKey == "color" && ruleValue === item[1]) ||
           (ruleKey == "name" && ruleValue === item[2]))
            count++;
    })

    return count;
};
